Modules/@babylonjs/react-native/windows/BabylonReactNative/BabylonModule.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "NativeModules.h"
+
+namespace winrt::BabylonReactNative::implementation {
+    REACT_MODULE(BabylonModule, L"BabylonModule");
+    struct BabylonModule : std::enable_shared_from_this<BabylonModule>
+    {
+        REACT_INIT(Initialize);
+        void Initialize(const winrt::Microsoft::ReactNative::ReactContext& reactContext) noexcept;
+
+        REACT_METHOD(CustomInitialize, L"initialize");
+        void CustomInitialize(const winrt::Microsoft::ReactNative::ReactPromise<bool>& result) noexcept;
+
+        BabylonModule();

```

Comment:
If these have empty implementations, can they be omitted?

Reply:
yes they can